zimui/src/components/playlist/PlaylistGrid.vue

```diff
@@ -8,23 +9,45 @@ const { mdAndDown } = useDisplay()
 const props = defineProps<{
   playlists: PlaylistPreview[]
 }>()
+
+const items = computed(() => props.playlists.slice(0, 48))
+
+const loadMoreItems = async () => {
+  return new Promise<PlaylistPreview[]>((resolve) => {
+    setTimeout(() => {
+      resolve(props.playlists.slice(items.value.length, items.value.length + 12))
+    }, 100)
+  })
+}
+
+const load = async ({ done }: { done: (status: 'ok' | 'empty') => void }) => {
+  const res = await loadMoreItems()
```

Comment:
Rename `res` to something more meaningful (`moreItems` / `nextItems` / ?)

Reply:
Changed it in [d1930a0](https://github.com/openzim/youtube/pull/289/commits/d1930a05040e8a51b581c2be289d366f87a93806)